CHANGELOG.md

```diff
@@ -32,6 +32,8 @@
 
 ### Added
 
+- [#3321](https://github.com/ChainSafe/forest/issues/3321): Support for random
+  file access in car-backend stores.
```

Comment:
We also used random access before this PR. The change is for parallel access to the car-backed stores.

```suggestion
- [#3321](https://github.com/ChainSafe/forest/issues/3321): Support for multi-threaded car-backed block stores.
```